module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2013,12 +2014,26 @@ protected function getItemStatusesForBiblio($id, $patron = null)
                 $duedate = null;
             }
 
+            $extraStatusInformation = [];
+            if ($transit = $avail['unavailabilities']['Item::Transfer'] ?? null) {
+                if (isset($transit['to_library'])) {
+                    $status = 'transit_to';
```

Comment:
I'm a bit torn on this. On one hand I don't really like it that it's the ILS driver's responsibility to choose the string to display, but on the other hand it's the most flexible approach. But if we know what we want to display, we could just set the final string as the status.

Reply:
Doing things this way (a translation string plus token array) instead of building the whole string here at least makes it a bit more abstract which would simplify some upstream customization scenarios...